src/containers/IntentService.jsx

```diff
@@ -36,23 +37,32 @@ export default class IntentService extends Component {
           disableSuccessTimeout: !!data.disableSuccessTimeout
         })
 
-        if (!data || !data.slug) {
+        if (!data) {
           throw new Error('Unexpected data from intent')
         }
 
-        return this.store.fetchKonnectorInfos(data.slug)
+        if (data.slug) {
+          return this.store.fetchKonnectorInfos(data.slug)
+        } else if (data.dataType) {
+          return this.store.findByDataType(data.dataType)
+        }
       })
-      .then(konnector => {
-        if (!konnector) {
+      .then(storeData => {
```

Comment:
I feel that `storeData` is a little bit too abstract, we should replace it by `konnectors`,  what do you think ?